packages/jest/package.json

```diff
@@ -41,7 +41,7 @@
     "@types/jest": "^27.0.3",
     "dtslint": "^4.2.1",
     "enzyme-to-json": "^3.6.1",
-    "preact": "^8.2.9",
+    "preact": "^10.6.6",
```

Comment:
could you recheck if Preact is used anywhere? I think this might be leftover from Emotion 9